### PR TITLE
Add CentOS/RH am-devtools setup

### LINKED ISSUE #168: Problem: There's no dev-tools setup support for CentOS/RH
There's no dev-tools setup support for CentOS/RH the way ubuntu's is supported.

### PR DESCRIPTION
**Pull request by mamedin**
This PR will setup am-devtools on CentOS/RH.

The ronn application package is installed via rubygems.

It fixes #168.

### CODE REVIEW

File: tasks/am-devtools.yml [hunk 1]
```diff
@@ -12,6 +12,45 @@
...
+    - "graphviz"
+    - "graphviz-devel"
+    - "graphviz-python"
+    - "python2-pkgconfig"
+  when:
+    - ansible_distribution == "RedHat" or ansible_distribution == "CentOS"
+
+# Needed to install ronn on RH/CentOS
+- name: "Install psych 2.2.4 (RH/CentOS)"
+  command: "gem install psych -v 2.2.4"
```
Comment (scollazo) [inline tasks/am-devtools.yml]: The package version should be in a variable, so it's easier to change

File: tasks/am-devtools.yml [hunk 2]
```diff
@@ -12,6 +12,45 @@
...
+    state: "latest"
+  with_items:
+    - "rubygems"
+    - "ruby-devel"
+    - "graphviz"
+    - "graphviz-devel"
+    - "graphviz-python"
+    - "python2-pkgconfig"
+  when:
+    - ansible_distribution == "RedHat" or ansible_distribution == "CentOS"
```
Comment (scollazo) [inline tasks/am-devtools.yml]: We can also us ansible_os_family,  and use only one condition, as we do here:

use https://github.com/artefactual-labs/ansible-archivematica-src/blob/7f56580b66f53647be562f0c574f870b47816f54/tasks/common.yml#L11

**Review by scollazo — CHANGES_REQUESTED** [review]
Aside from comments inline, I think we should wrap ubuntu/redhat code in different "block" sections, as shown here https://github.com/artefactual-labs/ansible-archivematica-src/blob/stable/1.6.x/tasks/pipeline-osdeps-MCPClient.yml#L3

File: tasks/am-devtools.yml (hunk 1 above)
Comment (sevein) [reply tasks/am-devtools.yml]: There's also the `gem` module, e.g.:

```
- gem:
  name: psych
  state: latest
```

**Comment by mamedin** [comment]
Thanks for the suggestions. I have just changed the PR, and it is ready for CR again.

**Review by sevein — APPROVED** [review]
LGTM

**Review by scollazo — APPROVED** [review]

**Review by scollazo — CHANGES_REQUESTED** [review]
Can you address my concerns and wrap the code in block: sections?

**Comment by mamedin** [comment]
Yes Santi, I forgot it. I'll change again to add blocks. Thanks for reminding me!!

**Comment by mamedin** [comment]
Hi, I added the block for every os_family.

**Review by scollazo — APPROVED** [review]
:+1: , LGTM!

**Pull request merged**